### ISSUE TITLE
Commit for pmbootstrab

### ISSUE THREAD

**Issue by tobwegit** [robante15/mia2_pmaports#2]
Dear Robante15,

I'd like to install postmarketOS with Kde Plasma for Phone on my Xiaomi Mi A2. But there is no selection for xiaomi-jasmine_sprout. After contact the IRC chat of postmarketOS, the other suggested that you can make a commit for that.

Could you please do a commit for xiaomi-jasmine_sprout (-:

Kind regards,
Tobias

**Comment by Nokse22** [comment]
If you do pmbootstrap init it should generate a folder, inside there is another folder named testing, if you paste the device and linux folder contained here the option should be available.

**Comment by tobwegit** [comment]
Hello Nokse22,
copying the folders in the testing folder worked for the jasmine_sprout selection. But there is no KDE Plasma Phone option in the User interface selection for jasmine_sprout:

NOTE: 6 UIs are hidden because "deviceinfo_gpu_accelerated" is not set (see https://postmarketos.org/deviceinfo).

Is this my end )-:


Tobias

**Comment by Nokse22** [comment]
Yes, GPU acceleration doesn't work and for that matter many other features too as it says on the Wiki page. Without gpu acceleration some UIs won't work. I successfully installed gnome, but it's extremely limited because even wifi doesn't work.

**Comment by robante15** [comment]
I have not continued testing pmos because the Mi A2 is still my daily use phone (I am still in love with this phone, I have not found one that I like enough to replace this one)

But they could look at branches of other devices to see if they have managed to activate other features. I remember that at the time I was close to making the modem and sound work, but it was a while ago, I remember other similar devices that were the Redmi Note 7
